privacy_bot/find_policies.py

```diff
@@ -131,35 +98,37 @@ def main():
 
     # Fetch data
     if urls:
-        print(">> Privacy Bot",                 file=sys.stderr)
-        print('-' * 80,                         file=sys.stderr)
-        print("Processing %s urls" % len(urls), file=sys.stderr)
-        print("Number of jobs: %s" % jobs,      file=sys.stderr)
-        print('-' * 80,                         file=sys.stderr)
-        print('',                               file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('{0:35}'.format('Initializing Privacy Bot'), file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('{0:35}{1:7}'.format('Urls to Process: ', len(urls)), file=sys.stderr)
+        print('{0:35}{1:7}'.format('Number of Jobs: ', jobs), file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('', file=sys.stderr)
```

Comment:
It slightly compromises the readability of the code, but it improves the readability of the output.